app/handlers/signup_start.rb

```diff
@@ -51,6 +56,27 @@ def handle
     outputs.signup_state = new_signup_state
   end
 
+  def confirm_trusted_account(signup_state)
```

Comment:
I have a non-firm feeling that this code should be somewhere else :-) like on its own somewhere.... but it isn't clear in my brain.

Like this is somehow a "calling app said just make a user and don't get a password" routine or something, and maybe not called from `SignupStart`... dunno

Reply:
Yes, I've moved the trusted_data storage onto signup-state, and have added logic where needed to consult it, rather than creating users in multiple spots